pipelinewise/cli/tap_properties.py

```diff
@@ -46,6 +46,43 @@ def generate_tap_s3_csv_to_table_mappings(tap):
     return s3_csv_tables
 
 
+def generate_tables_list(tap, as_string=False):
+    """
+    Generating table names from tap YAMLs in <SCHEMA_NAME>.<TABLE_NAME> format.
+    Some tap configurations are required to specify list of tables.
+
+    Example output as list:
+        "tables": [
+            "MY_SCHEMA.MY_TABLE_1",
+            "MY_SCHEMA.MY_TABLE_2"
+        ]
+
+    Example output as comma separated string that compatible with tap-snowflake tap
+        "tables": "MY_SCHEMA.MY_TABLE_1,MY_SCHEMA.MY_TABLE_2"
+    """
+    tables_list = []
+
+    # Using the input tap YAML we can generate the required config.json
+    schemas = tap.get('schemas', []) if tap else None
+    if schemas:
+        for schema in schemas:
+            tables = schema.get('tables', [])
+            for table in tables:
+                schema_name = schema.get('source_schema')
```

Comment:
`source_schema` and `table_name` are required in the tap config, thus it's better to use brakets `schema['source_schema']` instead of get, we don't want the software to continue executing with None values without any alert in case these properties weren't provided.

Reply:
thanks, fixed by 40b26e5cde5cc35516380045006bfe35e3706e64